src/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/General/UnoverriddenApis.cs

```diff
@@ -0,0 +1,234 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//
+// Why this file exists:
```

Comment:
I plan on removing this file; I don't think the intellisense issues are great enough to have a different between release and debug here.